packages/components/src/Loading/LoadingParagraph/LoadingParagraph.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./LoadingParagraph.module.scss"
+
+export type LoadingParagraphProps = {
+  isAnimated?: boolean
+  isCentred?: boolean
+  isReversed?: boolean
+  isInline?: boolean
+  noBottomMargin?: boolean
+  isLink?: boolean
+  inheritBaseline?: boolean
+  /**
+   * Width as a percentage.
+   */
+  width?: number
+} & OverrideClassName<HTMLAttributes<HTMLDivElement>>
+
+/**
+ * {@link https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082060062/Loading+Skeleton Guidance} |
+ * {@link https://cultureamp.design/?path=/docs/components-loading-states-loadingparagraph--docs Storybook}
+ */
+export const LoadingParagraph = ({
+  isAnimated,
+  isCentred,
+  isReversed,
+  isInline,
+  noBottomMargin,
+  width,
+  isLink,
+  inheritBaseline,
```

Comment:
Do you think there's any value to have a story for this prop? As far as I can tell it just adds a `position: static` to the div, which I would assume it has by default? Just wander what this actually does 🤔

Reply:
You ask a good question. I also had no idea what the intent is. I temporarily made a story, then it looked absolutely no different. Given I had no explanation for its existence, I opted to not add a story so people hopefully don't ask about it 😬

Reply:
🤞 here's hoping! I would've assume in the name it would be to match to `vertical-align: baseline` but that's just a stab in the dark 🤷 It's probably not of any consequence in this migration but we may be able to remove it later if we find no one uses it